convention-develocity-maven-extension/examples/maven_3.6/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.myorg</groupId>
+    <artifactId>maven-sample</artifactId>
+    <version>1.0</version>
+
+    <name>Convention Develocity Maven Extension Example</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Is this actually required, or could it be removed to further simplify the sample?

Reply:
It is not _required_, but without it there was a warning related to encoding printed to the console. I am fine with either leaving it or removing it.

Reply:
Ah yeah, leave it then. NO need to have warnings.